src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,166 @@
+package duke;
+
+import duke.tasks.Deadline;
+import duke.tasks.Todo;
+import duke.tasks.Task;
+import duke.tasks.Event;
+
+import duke.exception.DukeException;
+import duke.exception.EmptyCommandException;
+import duke.exception.InvalidCommandException;
+import duke.exception.InvalidIndexException;
+
+import java.util.Scanner;
+
+public class Duke {
```

Comment:
Perhaps you can consider moving `Ui` and `Command` code to separate classes to better apply OOP principles?